packages/graphql/src/datapuntDb.js

```diff
@@ -0,0 +1,24 @@
+const debug = require("debug")("datapuntDb");
```

Comment:
Shall we rename all datapunt to amsterdam? Including this file

Reply:
I'll rename this to database because (for now) we only have one.